src/base/rss/rssarticle.h

```diff
@@ -49,19 +49,19 @@ namespace Rss
         Q_OBJECT
 
     public:
-        Article(Feed *parent, const QString &guid);
+        Article(Feed *parent);
 
         // Accessors
-        bool hasAttachment() const;
-        const QString &guid() const;
```

Comment:
I still switched the `QString &` with `QString`, because otherwise I'd get
warning: C4172: returning address of local variable or temporary

should it be a `QString &` anyway?